content/projects/data-science-specific/statistical-thinking/_index.md

```diff
@@ -71,4 +71,44 @@ Import the [MDG data](MDG_Export_20191227.csv) to a Jupyter Notebook. You will n
 - Interpret the size and direction of the correlation statistic.
 - Is the relationship statistically significant? Report the appropriate statistic(s) to support your answer.
 
-13. Calculate the Spearman Rank-Order Correlation Coefficient. This test only looks at the _order_ of the categories, not the _values_. The Spearman Rank-Order Coefficient is therefore not influenced by non-normality of variables or outliers. How do the results of this test compare the results of the Pearson's correlation?
\ No newline at end of file
+13. Calculate the Spearman Rank-Order Correlation Coefficient. This test only looks at the _order_ of the categories, not the _values_. The Spearman Rank-Order Coefficient is therefore not influenced by non-normality of variables or outliers. How do the results of this test compare the results of the Pearson's correlation?
+
+## Instructions for reviewer
+
+1. A learner should be able to find any problems in the dataset without printing out the entire dataset like
+   one would do in an Excel spreadsheet for example.  One thing to check for is changing the way pandas prints
+   a dataset, things such as pd.set_option( ).
+
+2. Learners should understand the difference between a missing value and a NaN value.  It seems that students use
+   the terminology interchangeably.
+
+3. Learners need to be able to make the correct deductions following commands such as .info( ) and .describe( ). 
+   Sometimes learners will use these commands since they know it is required of them, but they are not entirely 
+   comfortable with what is presented to them by these commands.
+
+4. How do learners answer basic questions such as 'How many countries are represented'.  If it is a case of endless
+   'for' loops and 'if' statements then the recruit is not comfortable with the basic functionality which pandas or
+   numpy provides.
+
+5. Does the learner clean the dataframe, impute missing values and set the dataframe in a neat order at the beginning
+   of the project.  If not, the learner will continue to create sub-dataframes for each question to follow.  This is
+   bad practice.
```

Comment:
There is some odd spacing going on where there are sentence brakes on new lines like 
> of the project. 

If it doesn't affect how `hugo` renders the text on the site then it doesn't matter much but its some thing you should check.
```suggestion
5. Does the learner clean the dataframe, impute missing values and set the dataframe in a neat order at the beginning of the project.  If not, the learner will continue to create sub-dataframes for each question to follow.  This is bad practice.
```